src/editImageModalIndex.jsx

```diff
@@ -10,6 +10,7 @@ if ((typeof window !== 'undefined' && !window._babelPolyfill) ||
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { IntlProvider } from 'react-intl';
+import 'custom-event-polyfill'; // needed for IE11
```

Comment:
This is kind of unrelated to this PR, because I noticed that the new modal is broken in IE11 on master right now. This import fixes it though.

Reply:
👍